packages/angular/src/schematics/ngrx/ngrx.spec.ts

```diff
@@ -207,10 +230,38 @@ describe('ngrx', () => {
 
     const appModule = getFileContent(tree, '/apps/myapp/src/app/app.module.ts');
     expect(appModule).not.toContain('StoreModule');
-    expect(appModule).not.toContain(
-      '!environment.production ? [storeFreeze] : []'
+    expect(appModule).not.toContain('!environment.production ? [] : []');
+
+    [
```

Comment:
Should we also test for the existence of `state.models.ts` if the `creators` option is chosen?

Reply:
@brandonroberts What are your thoughts about adding a bit to test for `state.models.ts` as mentioned above?